packages/taskwarrior.rb

```diff
@@ -0,0 +1,63 @@
+require 'buildsystems/cmake'
+
+class Taskwarrior < CMake
+  description 'Free and Open Source Software that manages your TODO list from the command line.'
+  homepage 'https://taskwarrior.org/'
+  version '3.2.0'
+  license 'MIT'
+  compatibility 'all'
+  source_url "https://github.com/GothenburgBitFactory/taskwarrior/releases/download/v#{version}/task-#{version}.tar.gz"
+  source_sha256 '1a543ba373a319f924a2242c8e0db9dac2691c4cf654c2d0c6827a06b3b32592'
+  binary_compression 'tar.zst'
+
+  binary_sha256({
+    aarch64: '3a36a5a01e828bff4e7790187ca959fc876135f648678b62a790ee73dccca25a',
+     armv7l: '3a36a5a01e828bff4e7790187ca959fc876135f648678b62a790ee73dccca25a',
+       i686: '323138d15575d05c63b4c342bb0eae49257250a31500319f25e28e11df9642c8',
+     x86_64: 'f56cfb583bcb929406da7ac7724c0ff712b883890584b6fc0205bfccd6bb07fb'
+  })
+
+  depends_on 'rust' => :build
+  depends_on 'gcc_lib' # R
+  depends_on 'glibc' # R
+  depends_on 'util_linux' # R
+
+  def self.postinstall
+    ExitMessage.add "\nType 'man task' for help to get started.\n"
+  end
+
+  def self.postremove
+    taskrc = "#{HOME}/.taskrc"
+    if File.exist? taskrc
```

Comment:
We have a postinstall remove macro now!

Reply:
Does it work for files and directories?

Reply:
I believe so? Do you want to check?

Reply:
I examined remove.rb and didn't notice any special logic to handle the cleanup.  Also, many packages still have logic similar to this.

Reply:
See this in `package.rb`:
```ruby
  def self.agree_to_remove(config_object = nil)
    if File.file? config_object
      identifier = 'file'
    elsif File.directory? config_object
      identifier = 'directory'
    else
      puts "Cannot identify #{config_object}.".lightred
      return
    end
    if agree_default_no("Would you like to remove the config #{identifier}: #{config_object}")
      FileUtils.rm_rf config_object
      puts "#{config_object} removed.".lightgreen
    else
      puts "#{config_object} saved.".lightgreen
    end
  end
```

In `ruby_pry.rb`:
```ruby
  def self.postremove
    Package.agree_to_remove("#{HOME}/.pryrc")
  end
```

In `stellarium.rb`:
```ruby
  def self.postremove
    Package.agree_to_remove("#{HOME}/.stellarium")
  end
```

Is that not cleaner?

Reply:
Gotcha, didn't notice that.